client/dashboard/data/site-logs.ts

```diff
@@ -0,0 +1,142 @@
+import wpcom from 'calypso/lib/wp';
+
+export interface SiteLogsAPIResponse {
```

Comment:
I think we now don't need to export this type, and all the `*FromEndpoint` types.